python/docs/source/reference/pyspark.errors.rst

```diff
@@ -27,3 +27,14 @@ Errors
 
     PySparkException.getErrorClass
     PySparkException.getMessageParameters
+    CapturedException.getErrorClass
+    CapturedException.getMessageParameters
+    CapturedException.getSqlState
```

Comment:
We should probably separate methods vs classes. I don't mind if you plan to do this separately.

BTW, can we just document three methods?

```
    PySparkException.getErrorClass
    PySparkException.getMessageParameters
    PySparkException.getSqlState
```

Reply:
`PySparkException` has no `getSqlState` method, because SQLSTATE is only exists in errors from JVM side, which is captured by `CapturedException`

Thins maybe do you mean

```
    PySparkException.getErrorClass
    PySparkException.getMessageParameters
    CapturedException.getSqlState
```
?

Reply:
Just separated the class / methods and list only three methods below:

```
    PySparkException.getErrorClass
    PySparkException.getMessageParameters
    CapturedException.getSqlState
```

Reply:
Can we have `PySparkException.getSqlState` to match w/ JVM? By default, we could return `None` or something like undefined SQL state

Reply:
Got it.
I think we can return `None` as default since `getSqlState` returns `null` when SQLSTATE not defined in JVM side.

Reply:
Updated, thanks!